Core/Utilities/MySQL.py

```diff
@@ -575,15 +575,12 @@ def _connect(self):
     if self._connected:
       return S_OK()
 
-    # self.log.debug('_connect: Attempting to access DB',
-    #               '[%s@%s] by user %s' %
-    #               (self.__dbName, self.__hostName, self.__userName))
-    try:
-      # self.log.debug('_connect: Connected.')
-      self._connected = True
-      return S_OK()
-    except Exception as x:
-      return self._except('_connect', x, 'Could not connect to DB.')
+    # Test the connection to the DB
+    retDict = self._getConnection(retries=3)
```

Comment:
Should it use `MAXCONNECTRETRY`?

Reply:
I changed for `MAXCONNECTRETRY`.
It is just going to be a bit long if we have `Connection Timeout` multiple time but this is okay I guess.